protoc-gen-orm/testdata/Makefile

```diff
@@ -0,0 +1,11 @@
+# Variables
+PROJECT = ormpb
+REPO_ROOT = ${GOPATH}/src/github.com/gophersbd
+ROOT = ${REPO_ROOT}/${PROJECT}
+
+regenerate:
+	@echo WARNING! THIS RULE IS PROBABLY NOT RIGHT FOR YOUR INSTALLATION
+	@prototool compile message_options.proto
+	@protoc --go_out=../../. -I${ROOT} ${ROOT}/protoc-gen-orm/testdata/message_options.proto
```

Comment:
This should be like bellow:
`@protoc --go_out=plugins=orm:../../. -I${ROOT} ${ROOT}/protoc-gen-orm/testdata/message_options.proto`

Failed to make it works. Added another command to generate orm.